services/s3/src/main/java/software/amazon/awssdk/services/s3/S3Configuration.java

```diff
@@ -50,7 +50,7 @@
     /**
      * S3 payload checksum validation is by default enabled;
      */
-    private static final boolean DEFAULT_CHECKSUM_VALIDATION_ENABLED = true;
+    private static final boolean DEFAULT_CHECKSUM_VALIDATION_ENABLED = false;
```

Comment:
Hmm, I think we should keep checksum enabled by default.

Reply:
+1; what are the implications of changing the default apart from the presigner?

Reply:
The implication is that the SDK will no longer do data integrity validation for get and put object by default, which is not what we want :)